src/core/createCookieTransfer.js

```diff
@@ -67,11 +67,18 @@ export default ({ cookieJar, orgId, apexDomain }) => {
     responseToCookies(response) {
       response.getPayloadsByType(STATE_STORE_HANDLE_TYPE).forEach(stateItem => {
         const options = { domain: apexDomain };
+        const sameSite = stateItem.sameSite && stateItem.sameSite.toLowerCase();
 
         if (stateItem.maxAge !== undefined) {
           // cookieJar expects "expires" in days
           options.expires = convertTimes(SECOND, DAY, stateItem.maxAge);
         }
+        if (sameSite !== undefined) {
+          options.sameSite = sameSite;
+        }
+        if (sameSite === "none" || stateItem.secure === true) {
```

Comment:
Should we add a comment that we don't really expect the Edge to set the `secure` attribute, so that's why we're setting it? Just for future eyes :)